skills/findChannel.js

```diff
@@ -0,0 +1,21 @@
+class Project{
+  constructor(channel){
+    this.channel_id = channel;
+  }
+
+  static findByChannel(channel){
+    return new Promise(function(resolve,reject){
+      let projectChannel = new Project(channel)
+      resolve(projectChannel);
+    });
+
+  }
+
+}
+
+async function getNewProject(channelID){
```

Comment:
Let’s remove this since it’s not part of the app

Reply:
ok